package.json

```diff
@@ -9,11 +9,15 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
+    "i18next-http-backend": "^1.2.2",
     "next": "10.1.3",
+    "next-i18next": "^7.0.1",
+    "primeflex": "^2.0.0",
+    "primeicons": "^4.1.0",
+    "primereact": "^6.3.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "next-i18next": "^7.0.1",
-    "i18next-http-backend": "^1.2.2"
```

Comment:
this is the line that is showing deleted and added in the same file. I have not added it. because of the removed ',' this line is being considered as new.